apps/web/src/components/Table/ApiManagedTable/useFilterOptions.tsx

```diff
@@ -140,6 +141,21 @@ export default function useFilterOptions(enableEducationType = false) {
         label: intl.formatMessage(getCandidateSuspendedFilterStatus(value)),
       }),
     ),
+    roles: filterRes.data?.roles
+      ?.filter(notEmpty)
+      // custom selection of what options are desired in the input
+      .filter(
+        (role) =>
+          role?.name === ROLE_NAME.PlatformAdmin ||
+          role?.name === ROLE_NAME.PoolOperator ||
+          role?.name === ROLE_NAME.RequestResponder,
+      )
+      .map((role) => ({
+        value: role.id,
+        label:
+          (role?.displayName && role.displayName[locale]) ??
+          intl.formatMessage(commonMessages.nameNotLoaded),
+      })),
```

Comment:
I know the ticket said we want to filter by these roles but, is there any downside to allowing admins the ability to filter by any role? 🤔